ROMFS/px4fmu_common/init.d/50002_traxxas_stampede_2wd

```diff
@@ -25,7 +25,7 @@ then
 
 	param set NAV_ACC_RAD 0.5
 
-	param set MIS_LTRMIN_ALT 0.01
+	param set NAV_LTRMIN_ALT 0.01
```

Comment:
This change is evil and will break setups. It's not worth the "cleanup".

Reply:
Alright, I'll revert this part.
The line between navigator and mission parameters is pretty blurred at this point.